pyodide-build/pyodide_build/common.py

```diff
@@ -348,18 +350,68 @@ def search_pyodide_root(curdir: str | Path, *, max_depth: int = 5) -> Path:
     )
 
 
-def init_environment() -> None:
+def init_environment(*, quiet: bool = False) -> None:
+    """
+    Initialize Pyodide build environment.
+    This function needs to be called before any other Pyodide build functions.
+    """
     if os.environ.get("__LOADED_PYODIDE_ENV"):
         return
+
     os.environ["__LOADED_PYODIDE_ENV"] = "1"
+
+    _set_pyodide_root(quiet=quiet)
```

Comment:
Is `_set_pyodide_root` ever called from anywhere else? Or is the split purely for readability? Might be good to mention.

Reply:
It's purely for readability and `_set_pyodide_root` is not supposed to be called in other places. I wanted to split the build environment initialization into several steps. Will add some comments.